### PR TITLE
ACI Tenant EP Span Src Grp: Update module to use new URL method

### PR DESCRIPTION
**Pull request by jmcgill298**

### CODE REVIEW

File: library/aci_tenant_span_src_group.py [hunk 1]
```diff
@@ -29,174 +29,104 @@
...
-            - Password used to login to the switch
-        required: true
-    protocol:
-        description:
-            - Dictates connection protocol to use
-        default: https
-        choices: ['http', 'https']
+  admin_state:
+    description:
+    - Enable or Disable the span sources.
```
Comment (dagwieers) [inline library/aci_tenant_span_src_group.py]: Not sure why "Disable" is title-case ?

File: library/aci_tenant_span_src_group.py [hunk 2]
```diff
@@ -29,174 +29,104 @@
...
+    choices: [ enabled, disabled ]
+    default: enabled
+  description:
+    description:
+    - The Description for Span source group.
```
Comment (dagwieers) [inline library/aci_tenant_span_src_group.py]: Same here for Description.

File: library/aci_tenant_span_src_group.py (hunk 1 above)
Comment (jmcgill298) [inline library/aci_tenant_span_src_group.py]: I titled it for symmetry; can make it lower-case

File: library/aci_tenant_span_src_group.py (hunk 2 above)
Comment (jmcgill298) [inline library/aci_tenant_span_src_group.py]: should be lower-case

**Pull request merged**